src/python/nimbusml/pipeline.py

```diff
@@ -1771,6 +1773,13 @@ def get_feature_contributions(self, X, top=10, bottom=10, verbose=0,
             data="$data",
             predictor_model="$predictor_model",
             scored_data="$scoredvectordata")
+        
+        if isinstance(X, csr_matrix):
+            X = X.astype(np.float32)
```

Comment:
> X = X.astype(np.float32) [](start = 11, length = 25)

Only Single type can be run as Feature vector input so the conversion has to happen anyway either in ML.NET or python.